research/object_detection/protos/post_processing.proto

```diff
@@ -39,6 +39,9 @@ message BatchNonMaxSuppression {
 
   // Soft NMS sigma parameter; Bodla et al, https://arxiv.org/abs/1704.04503)
   optional float soft_nms_sigma = 9 [default = 0.0];
+
+  // Whether to use tf.image.combined_non_max_suppression
```

Comment:
Please add period to the end of your comments (same for your other changes in this PR as well).

Reply:
Done.